pkg/plugin/loader/loader_test.go

```diff
@@ -235,7 +235,7 @@ func TestFilterList(t *testing.T) {
 		{Name: "test3"},
 	}
 
-	expected := []*manifest.Manifest{definitions[0]}
+	expected := []manifest.Manifest{definitions[0]}
```

Comment:
In a bunch of situations, we were using pointers when I don't think it was necessary (and we had gone from a non-pointer plugin.Definition to a pointer to a manifest.Manifest).